### PR TITLE
Silences log output when in Test environment and with Webhook verifiers

### PR DESCRIPTION
**Pull request by dthyresson**
Addresses issue https://github.com/redwoodjs/redwood/issues/2246

This PR:

* Sets the default log level when in Test to 'silent' - which will not produce any log output.
* One can elect to turn it on explicitly in the test by setting the log level
* Makes the Webhook verifier warnings less chatty when running in dev/prod (yet still warns when using skip and secret key)
* However, it suppresses the `console.warn` in those verifier tests to make jest test output cleaner

### CODE REVIEW

**Comment by dthyresson** [comment]
Note this does not need to be included in 0.32 but @dac09 you can make that call.

**Comment by dac09** [comment]
Let's keep this out for this release DT if its okay with you?

**Comment by dthyresson** [comment]
👍

File: packages/api/src/auth/verifiers/__tests__/secretKeyVerifier.test.ts [hunk 1]
```diff
@@ -5,6 +5,14 @@ const secret = 'MY_VOICE_IS_MY_PASSPORT_VERIFY_ME'
 
 const { sign, verify } = createVerifier('secretKeyVerifier')
 
+beforeEach(() => {
+  jest.spyOn(console, 'warn').mockImplementation(jest.fn())
+})
+
+afterEach(() => {
+  jest.spyOn(console, 'warn').mockRestore()
+})
```
Comment (peterp) [inline packages/api/src/auth/verifiers/__tests__/secretKeyVerifier.test.ts:14]: I'm  a bit concerned  about doing this - isn't there a way to disable the logger during tests instead?

Comment (peterp) [reply packages/api/src/auth/verifiers/__tests__/secretKeyVerifier.test.ts:14]: The reason I'm concerned is that we might silence output that's requires a user's attention.

File: packages/api/src/auth/verifiers/jwtVerifier.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ export const verifySignature = ({
 
 /**
  *
- * JWT Payload Verfifier
+ * JWT Payload Verifier
```
Comment (peterp) [inline packages/api/src/auth/verifiers/jwtVerifier.ts:79]: Michelle Verfifier is my favourite actor

**Review by peterp — APPROVED** [review]
LGTM

**Comment by peterp** [comment]
@dac09 This is such a minuscule change - I don't think it'll cause any concerns for v0.32.0.

**Comment by dac09** [comment]
You got it @peterp!

**Comment by dthyresson** [comment]
Thanks!

-- david

**Pull request merged**